convert2rhel/rhelvariant.py

```diff
@@ -18,11 +18,16 @@
 import os
 import re
 import logging
+
 from convert2rhel import utils
 
 _supported_variants = None
 
 
+def check_rhn_classic():
+    return os.path.isfile("/etc/sysconfig/rhn/systemid")
+
+
```

Comment:
This can be a private function, since we won't use it elsewhere in the app. That way we don't need to write tests for it and can test in `determine_rhel_variant()` instead

Convention would be `_check_rhn_classic()`

Reply:
I think that in general, we should write unit tests for any function regardless of being private or public. We want to ensure that each function does what it's supposed to.
However, in this case, a unit test wouldn't make much sense - you would be just testing a standard library.

Reply:
Agree. Changing to private function, and removing the unit test